lib/router.js

```diff
@@ -633,3 +634,38 @@ Router.prototype.toString = function toString() {
 
     return (str);
 };
+
+
+/**
+ * Return information about the routes registered in the router.
+ * @public
+ * @returns {object} The routes in the router.
+ */
+Router.prototype.getRoutes = function getRoutes() {
```

Comment:
Oh I see, you're using this in server's debug. Still think we could use a better name, though.